apps/frontend/src/pages/chat/libs/components/chat-message/chat-message.tsx

```diff
@@ -0,0 +1,81 @@
+import defaultAvatar from "~/assets/img/default-avatar.png";
+import { Icon } from "~/libs/components/components.js";
+import { getValidClassNames } from "~/libs/helpers/helpers.js";
+import { useAppSelector } from "~/libs/hooks/hooks.js";
+import { type UserDto, type ValueOf } from "~/libs/types/types.js";
+import {
+	ChatMessageAuthor,
+	ChatMessageType,
+	type TaskMessage,
+	type TextMessage,
+} from "~/modules/chat/chat.js";
+
+import { TaskSuggestionsDisplay } from "./libs/components/components.js";
+import styles from "./styles.module.css";
+
+type Properties = {
+	author: ValueOf<typeof ChatMessageAuthor>;
+	payload: TaskMessage | TaskMessage[] | TextMessage;
+	type: ValueOf<typeof ChatMessageType>;
+};
+
+const ChatMessage: React.FC<Properties> = ({
+	author,
+	payload,
+	type,
+}: Properties) => {
+	const user = useAppSelector((state) => state.auth.user as UserDto);
+	const messageContainerStyle = getValidClassNames(
+		styles["message-container"],
+		author === "assistant"
+			? styles["message-container-assistant"]
+			: styles["message-container-user"],
```

Comment:
Let's move it to containerStyles variable

Reply:
It is already a part of a separate variable, I wouldn't create a new one just for one condition